src/behaviors/franka_circular_cartesian.cpp

```diff
@@ -0,0 +1,84 @@
+#include "inria_wbc/behaviors/franka_circular_cartesian.hpp"
+
+namespace inria_wbc {
+    namespace behaviors {
+
+        static Register<CircCartTraj> __franka_circular_cartesian_trajectory("circular-cartesian-trajectory");
+
+        CircCartTraj::CircCartTraj(const controller_ptr_t& controller) : Behavior(controller)
+        {
+            traj_index_=0.;
+            YAML::Node config =
+              YAML::LoadFile(controller_->params().sot_config_path)["BEHAVIOR"];
+
+            pitch_angle_ = config["pitch_angle"].as<float>();
+            radius_ = config["radius"].as<float>();
+            xyz_offset_ <<
```

Comment:
I suggest to use vectors in yaml (instead of x0: ...). This is what where use everywhere.

Reply:
changed to accept vector from yaml file for initial position instead of x_0, y_0 and z_0 separately